### PR TITLE
user defined types

### PR DESCRIPTION
**Pull request by scott-roepnack-zocdoc**
cc @Zocdoc/ci-devs

### CODE REVIEW

**Comment by scott-roepnack-zocdoc** [comment]
image

**Comment by scott-roepnack-zocdoc** [comment]
image

**Comment by scott-roepnack-zocdoc** [comment]
image

File: model/Models/Database.cs [hunk 1]
```diff
@@ -857,7 +862,42 @@ from INFORMATION_SCHEMA.TABLES
...
+            select
+                s.name as 'Type_Schema',
+                t.name as 'Type_Name',
+                tt.name as 'Base_Type_Name',
+                t.max_length as 'Max_Length',
+                t.is_nullable as 'Nullable'	
+            from sys.types t
+            inner join sys.schemas s on s.schema_id = t.schema_id
+            inner join sys.types tt on t.system_type_id = tt.user_type_id
+            where 1=1
```
Comment (colin-hanson-zocdoc) [inline model/Models/Database.cs]: 1=1? I assume you had that for easy query development?

**Comment by colin-hanson-zocdoc** [comment]
:+1:

**Pull request merged**